### PR TITLE
Update en.json

### PR DESCRIPTION
**Pull request by Tirokk**
> [erhardtstiefel](https://github.com/erhardtstiefel) **Authored by [erhardtstiefel](https://github.com/erhardtstiefel)**
_<time datetime="2019-12-07T15:27:56Z" title="Saturday, December 7th 2019, 4:27:56 pm +01:00">Dec 7, 2019</time>_
_Closed <time datetime="2019-12-11T10:32:52Z" title="Wednesday, December 11th 2019, 11:32:52 am +01:00">Dec 11, 2019</time>_
---

## 🍰 Pullrequest

> Form Title changed from 'Delete data' to 'Delete account'
> Describing message modified. 
> Warning! added
Bugfixes 01e

### Issues

modify template deleteUserAccount in DeleteData.vue

### CODE REVIEW

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2019-12-07T15:54:53Z" title="Saturday, December 7th 2019, 4:54:53 pm +01:00">Dec 7, 2019</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/2714/specs"></a> <b title="Passed">48</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/2714/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | 260878b79d |
| **Started** | Dec 7, 2019 3:48 PM |
| **Ended** | Dec 7, 2019 4:01 PM |
| **Duration** | 13:05 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Chromium 78  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/2714/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

File: webapp/locales/en.json [hunk 1]
```diff
@@ -269,11 +269,9 @@
       "name": "Download Data"
     },
     "deleteUserAccount": {
-      "name": "Delete data",
-      "contributionsCount": "Delete my {count} posts",
-      "commentedCount": "Delete my {count} comments",
```
Comment (Tirokk) [inline webapp/locales/en.json:274]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-07T16:07:17Z" title="Saturday, December 7th 2019, 5:07:17 pm +01:00">Dec 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -269,11 +269,9 @@
       "name": "Download Data"
     },
     "deleteUserAccount": {
-      "name": "Delete data",
-      "contributionsCount": "Delete my {count} posts",
-      "commentedCount": "Delete my {count} comments",
```


Bist du dir sicher, dass die Übersetzungen `contributionsCount` und `commentedCount` nicht mehr benötigt werden? Wenn ich den Befehl (s.u..) eingebe, dann erhalte ich einen Treffer im Code.

-----


Are you sure that `contributionsCount` and `commentedCount` are not needed anymore?

E.g. when I run:
```sh
git grep 'deleteUserAccount.commentedCount'
```

I see one search result in `webapp/components/DeleteData/DeleteData.vue`

Comment (Tirokk) [reply webapp/locales/en.json:274]: > [erhardtstiefel](https://github.com/erhardtstiefel) **Authored by [erhardtstiefel](https://github.com/erhardtstiefel)**
_<time datetime="2019-12-07T16:24:40Z" title="Saturday, December 7th 2019, 5:24:40 pm +01:00">Dec 7, 2019</time>_

---


Yes, I wrote another Pull Request 
Update DeleteData.vue #2447

**Comment by Tirokk** [comment]
> [erhardtstiefel](https://github.com/erhardtstiefel) **Authored by [erhardtstiefel](https://github.com/erhardtstiefel)**
_<time datetime="2019-12-11T10:32:52Z" title="Wednesday, December 11th 2019, 11:32:52 am +01:00">Dec 11, 2019</time>_

---


I closed this PR and created a new one #2491

**Review by Mogge — CHANGES_REQUESTED** [review]
Herzlichen Glückwunsch zu deinem ersten Pull Request!

bravocado

Jeder PR durchläuft ein "Review" bevor wir den code übernehmen. Schau dir mal meine Fragen an :point_down: 

----

Congrats for your first Pull Request!
Each Pull Request goues through a review cycle. See my question below.

**Pull request closed without merging**